doc/classes/TextureButton.xml

```diff
@@ -37,7 +37,7 @@
 			Texture to display when the mouse hovers the node.
 		</member>
 		<member name="texture_normal" type="Texture2D" setter="set_texture_normal" getter="get_texture_normal">
-			Texture to display by default, when the node is [b]not[/b] in the disabled, focused, hover or pressed state.
+			Texture to display by default, when the node is [b]not[/b] in the disabled, hover or pressed state.
```

Comment:
```suggestion
			Texture to display by default, when the node is [b]not[/b] in the disabled, hover or pressed state. This texture is still displayed in the focused state, with [member texture_focused] drawn on top.
```

Good spot, but I think we can explain it further instead of just removing.